doc/glossary.rst

```diff
@@ -5,27 +5,38 @@ Glossary
 
 .. glossary::
 
-    Reference ellipsoid
-        An ellipsoid with geometric and physical properties used to best
-        approaximate a given planetary body (the Earth or others).
-        See `Wikipedia <https://en.wikipedia.org/wiki/Reference_ellipsoid>`__.
-
+    Co-located grids.
+        The grid points are at exactly the same locations. It's important because otherwise we can't add or subtract values since they wouldn't be aligned. See `Normal gravity <https://www.fatiando.org/boule/latest/overview.html#normal-gravity>`__.
```

Comment:
```suggestion
    Co-located grids
        One or more grids that have grid points are at exactly the same locations. 
        This property is important when performing operations like adding and 
        subtracting grid values since they must be located at the same 
        geographical locations.
```

Reply:
@MGomezN the linking for glossary items usually goes the other way around. Instead of linking to the tutorial here, in the tutorial you can substitute the occurrence of "co-located grids" with `:term:`co-located grids` and it will automatically link to this glossary entry.

Reply:
Thanks for your patience and kind suggestions!